docs/nodes/nodes-library/core-nodes/SSH/README.md

```diff
@@ -0,0 +1,68 @@
+
+- ***Authentication:*** A dropdown list to choose between Password or Private Key authentication.
+- ***Resource:*** A dropdown list used to specify if you are executing a command or processing a file.
+- ***Operation:*** A dropdown list to select the action to be performed. When selecting **Command** as the ***Resource***, only **Execute** is available. For a **File** ***Resource*** you can select either **Upload** or **Download**.
+- ***Command:*** Only visible for **Command** resources. The command to execute on the remote machine.
+- ***Working Directory:*** Only visible for **Command** resources. The directory where the command should be executed.
+- ***Path:*** Only visible for **Download** operation on file resources. The path where the desired file is found.
+- ***Binary Property:*** Only visible for file resources. The name of the binary property which contains the data for the file to be uploaded.
+- ***Target Directory:*** Only visible for **Upload** operations on file resources. The directory to upload the file to. The name of the file does not need to be specified, it's taken from the binary data file name. To override this behavior, set the parameter **File Name** under options.
+
+
+## Example Usage
+
+This workflow allows you to remotely connect to your computer via SSH. This example usage workflow would use the following nodes.
+- [Start](../../core-nodes/Start/README.md)
+- [SSH]()
+
+The final workflow should look like the following image.
+
+![A workflow with the SSH node](./workflow.png)
+
+### 1. Start node
+
+The start node exists by default when you create a new workflow.
+
+### 2. SSH node
+
+1. From the SSH Credentials dropdown, select ***Create New*** and complete the following fields:
+    - ***Credentials Name***: Enter a descriptive name, here we used `ssh_demo`.
+    - ***Host***: Enter the IP address of the server you are connecting to. Here we are using `192.168.1.8` for a local Mac set up for SSH access.
+    - ***Port***: Enter the port to use for this connection. SSH used port 22 by default.
+    - ***User***: Enter the your username for this server.
+    - ***Password***: Enter your password for the provided username.
+    - ***Private Key***: Only available for Private Key authentication. Enter your SSH key.
+    - ***Passphrase***: Only available for Private Key authentication. Enter your passphrase.
+
+![SSH credentials](./ssh_credentials.png)
+
+2. Select ***Password*** from the ***Authentication*** dropdown list.
+3. Select ***Command*** from the ***Resource*** dropdown list.
+4. The ***Execute*** option is automatically selected as the ***Operation***.
+5. In the ***Command*** field enter the command to execute. Here we used `ls` just to see the local directory contents.
+6. In the ***Working Directory*** field you can enter where the command should be executed. Here we ran it in the root directory.
+7. Click on ***Execute Node*** to run the node.
+
+![Executed Workflow](./executed_ssh_workflow.png)
```

Comment:
The Alt text and the image name don't follow the standard we use.

Reply:
I don't believe there is any standard for this mentioned in the Contributing page - if it will be enforced that it should be included there.

Reply:
Ah yes. we don't have guidelines for this yet. But if you refer to other node documentation, you will get an idea of what I mean. The alt text is really helpful for people who use a screen reader.